CIP-1694/README.md

```diff
@@ -0,0 +1,1124 @@
+
+1. a normal state (i.e. a state of confidence)
+2. a state of no-confidence
+
+In a _state of no-confidence_, the current committee is no longer able to participate in governance actions
+and must be replaced before any governance actions can be ratified (see below).
+Any outstanding governance actions are dropped immediately after the protocol enters a state of no-confidence,
+and will not be enacted.
+
+The constitutional committee will use a hot and cold key setup, similar to the existing "genesis delegation certificate" mechanism.
+
+#### Initial constitutional committee
+
+The initial committee is not yet defined. In all likelihood, it will include some of Cardano's founding entities.
+
+#### Replacing the constitutional committee
+
+The constitutional committee can be replaced in one of two ways:
+
+* When in a **normal** state (i.e. a state of **confidence**), the committee can be replaced via a
+specific governance action (action 2 below) that requires the approval of both the **current constitutional committee** and the **DReps**.
+
+* When in a state of **no-confidence**, the committee can also be replaced via the same governance action (action 2 below),
+but this instead requires the approval of both the **SPOs** and the **DReps**.
+
+#### Size of the constitutional committee
+
+Unlike the Shelley governance design, the size of the constitutional committee is not fixed.
+It may be changed whenever a new committee is elected (action 2 below).
+Likewise, the committee _quorum_ (the number of committee `Yes` votes that are required to ratify governance actions) is not fixed and
+can also be varied by governance action 2.
+This gives a great deal of flexibility to the composition of the committee.
+
+#### Term limits
+
+
+The system will automatically enter a state of no-confidence when the term limit for the constitutional
+committee expires.  This limit is a governance protocol parameter, which specifies the maximum number of epochs
+during which the committee can ratify governance actions.  When the committee term limit expires, all governance
+actions will be dropped, and a new committee must be elected.
+
+The term limit will reset whenever governance action 2 is enacted, even if the same committee is re-elected
+and the quorum remains unchanged.  This allows Ada holders to confirm their confidence in the committee if they wish.
+Note that the term limit is calculated at the point the committee is elected.  Any change in the underlying protocol
+parameter only affects the term that applies to future committees.
+
+
+<!--------------------------- Constitutional committee ------------------------>
+<!---------------------------           DReps          ------------------------>
+
+### Delegated representatives (DReps)
+
+> **Warning**
+> CIP-1694 DReps **should not be conflated** with Project Catalyst DReps.
+
+<!-- TODO find another name that still points to liquid democracy. -->
+
+#### Pre-defined DReps
```

Comment:
I like this suggestion! I have the same worry with this proposal, and that solves it neatly. There are some variations of this idea: instead of having more `No Confidence` DReps, we could move everyone from `No Confidence` to `Abstain` once a new CC or a state of no confidence is enacted.